deployment/fedora-package-x64/package.sh

```diff
@@ -42,10 +46,8 @@ tar \
 --exclude='**/.nuget' \
 --exclude='*.deb' \
 --exclude='*.rpm' \
--Jcvf \
-"$package_temporary_dir/jellyfin-${VERSION}.tar.xz" \
--C "../.." \
-./ || true && GNU_TAR=0
+-zcf "$pkg_src_dir/jellyfin-${VERSION}.tar.gz" \
+-C "../.." ./ || GNU_TAR=0
```

Comment:
This might quit the script if run on any BSD like OS. That is why the true was there. If you switch it to a *.gz archive there you need to fix the workaround untarring in the tar call below as well.

Reply:
Can you also add an exception to the docker sudo group for macOS, Docker for Mac does not require sudo.

example:
```sh
if [[ "$OSTYPE" == "darwin"* ]]; then
```

Current value of `$OSTYPE` is `darwin18`.

Reply:
Added the macOS check, can you test it?
POSIX sh does not have `[[  ]]` for glob comparison (at least on FreeBSD), so i used `echo "$OSTYPE" | grep -q "darwin"` as a workaround.